### PR TITLE
#852 - vPool GUI work

### PR DESCRIPTION
**Pull request by kvanhijf**
Initial commit

### CODE REVIEW

**Comment by kvanhijf** [comment]
https://github.com/openvstorage/framework/pull/1130 --> openvstorage-2.7.5-rev.4273.d9bb7fb
https://github.com/openvstorage/framework-alba-plugin/pull/267 --> openvstorage-backend-1.7.5-rev.811.c1b83e6

File: ovs/lib/storagerouter.py [hunk 1]
```diff
@@ -103,91 +103,72 @@ def get_metadata(storagerouter_guid):
         storagerouter = StorageRouter(storagerouter_guid)
         client = SSHClient(storagerouter)
         ipaddresses = client.run("ip a | grep 'inet ' | sed 's/\s\s*/ /g' | cut -d ' ' -f 3 | cut -d '/' -f 1").strip().splitlines()
-        ipaddresses = [ipaddr.strip() for ipaddr in ipaddresses]
-        ipaddresses.remove('127.0.0.1')
+        ipaddresses = [ipaddr.strip() for ipaddr in ipaddresses if ipaddr.strip() != '127.0.0.1']
+        services_mds = ServiceTypeList.get_by_name(ServiceType.SERVICE_TYPES.MD_SERVER).services
+        services_arakoon = [service for service in ServiceTypeList.get_by_name(ServiceType.SERVICE_TYPES.ARAKOON).services if service.name != 'arakoon-ovsdb']
```
Comment (khenderick) [inline ovs/lib/storagerouter.py]: The name of the ovsdb should not be hardcoded, and you might just want to take internal arakoons into account.

File: webapps/api/backend/views/storagerouters.py [hunk 2]
```diff
@@ -215,7 +215,12 @@ def check_s3(self, host, port, accesskey, secretkey):
         :type accesskey: str
         :param secretkey: The secretkey to be used when validating the S3 endpoint
         :type secretkey: str
+        :param version: Client version
+        :type version: int
         """
+        if version >= 6:
```
Comment (khenderick) [inline webapps/api/backend/views/storagerouters.py]: You can let the `@load` decorator check that

File: webapps/frontend/app/viewmodels/wizards/addvpool/gather_vpool.js [hunk 3]
```diff
@@ -18,145 +18,146 @@ define([
...
+        self.data = data;
         self.shared = shared;
 
         // Handles
-        self.checkS3Handle            = undefined;
-        self.checkMtptHandle          = undefined;
-        self.checkMetadataHandle      = undefined;
-        self.fetchAlbaVPoolHandle     = undefined;
-        self.loadSRMetadataHandle     = undefined;
+        self.checkS3Handle = undefined;
```
Comment (khenderick) [inline webapps/frontend/app/viewmodels/wizards/addvpool/gather_vpool.js]: Alignments

File: webapps/frontend/app/views/site/storagerouter-detail.html [hunk 4]
```diff
@@ -181,7 +181,9 @@ <h4 data-i18n="ovs:storagerouters.detail.storagerouter_vpool_link"></h4>
                                                                       checked: $root.checkedVPoolGuids().contains(guid()),
                                                                       disable: true">
                                                 </td>
-                                                <td><span data-bind="text: name()"></span></td>
+                                                <td><a data-bind="attr: {href: $root.shared.routing.loadHash('vpool-detail', { guid: guid() })}">
```
Comment (khenderick) [inline webapps/frontend/app/views/site/storagerouter-detail.html]: Alignment

File: webapps/frontend/locales/en-US/ovs.json [hunk 5]
```diff
@@ -1,134 +1,61 @@
 {
-    "license": [
```
Comment (khenderick) [inline webapps/frontend/locales/en-US/ovs.json:2]: Call it `_license` and keep it on top

**Review by khenderick — CHANGES_REQUESTED** [review]

**Review by khenderick — APPROVED** [review]

**Pull request merged**